### DISCUSSION TITLE
Goodwe Iverter Lost Battery State of Charge

### DISCUSSION THREAD

**Discussion by oluapseven** [home-assistant/frontend#22263] [Configuration]
Hello everybody,

I have a weird issue with the Goodwe integration with Home Assistant.

One of the main Inverter Parameters is no longer available - Battery State of Charge.

The other main parameters are all ok, just this one became unnavailable...

Anyone had the same issue?
![imagem](https://github.com/user-attachments/assets/f14c31aa-307a-4c51-ac6d-e03ce00db5fe)
![imagem](https://github.com/user-attachments/assets/193836a8-6b21-4fca-86c9-ed053788a5b3)

**Comment by karwosts** [comment]
This isn't really a frontend question, you'd be better off asking in community forums, or maybe filing an issue in home-assistant/core.
